deeplake/htype.py

```diff
@@ -98,7 +101,95 @@ class htype:
     htype.INTRINSICS: {"dtype": "float32"},
     htype.POLYGON: {"dtype": "float32"},
     htype.MESH: {"sample_compression": "ply"},
-    htype.EMBEDDING: {},
+}
+
+HTYPE_CONVERSION_LHS = {htype.DEFAULT, htype.IMAGE}
+
+
+class constraints:
```

Comment:
I really like this class, can you write its name with capital letters? Also can we move all constants to the top of the class? I think they shouldn't be mixed with methods